ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml

```diff
@@ -65,8 +65,8 @@ tests:
     workflow: openshift-e2e-gcp
 - as: e2e-gcp-upgrade
   steps:
-    cluster_profile: gcp
-    workflow: openshift-upgrade-gcp
+    cluster_profile: azure4
```

Comment:
Huh mmnt, this one is not agnostic

Reply:
We may as well bite the bullet and rename it, if this is intentional?

Reply:
Right, let me do this

Reply:
Done